engine/CMakeLists.txt

```diff
@@ -32,8 +32,11 @@ if(UNIX AND NOT APPLE)
 	target_link_libraries(${PROJECT_NAME} PRIVATE ${X11_LIBRARIES} ${X11_Xxf86vm_LIB} GL)
 endif()
 
-target_precompiled_header(${PROJECT_NAME} source/include/quartz/core/QuartzPCH.hpp TYPE c++-header)
-set_target_properties(quartz-engine.pch PROPERTIES FOLDER quartz-dependencies)
+if (NOT UNIX)
```

Comment:
Do a

``if (MSVC)`` because it probably doesn't work w/ GCC even on Windows through MinGW.

Reply:
Done.